field_audit/models.py

```diff
@@ -135,15 +136,20 @@ class AuditEvent(models.Model):
     change_context = models.JSONField()
     is_create = models.BooleanField(default=False)
     is_delete = models.BooleanField(default=False)
+    is_bootstrap = models.BooleanField(default=False, db_index=True)
```

Comment:
Putting an index on a column with low cardinality probably does not provide much benefit, and may be worse than no index at all. [This article](https://distributedsystemsauthority.com/index-efficiency-and-maintenance-postgresql-12-high-performance-guide-part-5-12/) may be informative if you're not familiar with how cardinality affects indexes.

Reply:
Fixed in d21bc95 (forgot to leave this comment earlier).